pom.xml

```diff
@@ -597,6 +597,7 @@ limitations under the License.
                 <groupId>com.codahale.metrics</groupId>
                 <artifactId>metrics-graphite</artifactId>
                 <version>${metrics.version}</version>
+                <optional>true</optional>
```

Comment:
I don't follow - the `<version>` is there. It's presented as a pom variable, but it's defined.